pylint/checkers/classes/class_checker.py

```diff
@@ -613,7 +613,7 @@ def _has_same_layout_slots(slots, assigned_value):
     "E0241": (
         "Duplicate bases for class %r",
         "duplicate-bases",
-        "Used when a class has duplicate bases.",
+        "Duplicate use of base class in derived class are TypeError.",
```

Comment:
```suggestion
        "Duplicate use of base classes in derived classes raise TypeErrors.",
```

Reply:
Thank you, English is not my first language 🙏